### ISSUE TITLE
I'm having a cursed issue when trying to run Stitch. Any idea what this could be?

### ISSUE THREAD

**Issue by evolutionleo** [bscotch/stitch#36]
It was running fine before, but then it broke
```
Stitch ERROR 07:50:39 "git" �� ���� ����७��� ��� ���譥�
��������, �ᯮ��塞�� �ணࠬ��� ��� ������ 䠩���.
 >
StitchError: "git" �� ���� ����७��� ��� ���譥�

    at Gms2Storage.get workingDirIsClean [as workingDirIsClean] (C:\Users\evolu\Documents\Projects\Actual projects\GMS 2.3 Projects\UnnamedPlatformer\ImportingPipeline\node_modules\@bscotch\stitch\src\lib\Gms2Storage.ts:37:13)                                                                                                                                                                        src\lib\Gms2Storage.ts:37:13)
    at new Gms2Storage (C:\Users\evolu\Documents\Projects\Actual projects\GMS 2.3 Projects\UnnamedPlatformer\ImportingPipeline\node_modules\@bscotch\stitch\src\lib\Gms2Storage.ts:17:13)
    at new Gms2Project (C:\Users\evolu\Documents\Projects\Actual projects\GMS 2.3 Projects\UnnamedPlatformer\ImportingPipeline\node_modules\@bscotch\stitch\src\lib\Gms2Project.ts:159:22)
    at Object.<anonymous> (C:\Users\evolu\Documents\Projects\Actual projects\GMS 2.3 Projects\UnnamedPlatformer\ImportingPipeline\load.js:5:19)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1147:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at run (C:\Users\evolu\Documents\Projects\Actual projects\GMS 2.3 Projects\UnnamedPlatformer\ImportingPipeline\index.js:5:11)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by adam-coster** [comment]
Wow, that *is* cursed.

There code corresponding to this error is here: https://github.com/bscotch/stitch/blob/34a729a7576862aa2957ba4bcc549274f3ea5283/src/lib/Gms2Storage.ts#L31-L44

That error gets thrown when Stitch tries to talk to Git and gets an error. All of those symbols are from Stitch attempting to turn the error output into a human-readable string, but apparently whatever that data is isn't UTF8 text!

Specifically, Stitch is trying to run `git status` and for some reason getting an error code back from Git.

Some possibilities:

- Have you made any changes to your Git installation lately?
- If you try to run Git in your project (e.g. just with `git status`) does it work properly?
- How are you running Stitch? Via the command line or via code? If via the CLI, have you updated or changed terminals?
- Are you running Stitch in a VM, or something that doesn't have Git installed?

**Comment by evolutionleo** [comment]
Hmm, interesting...

1) I don't think I have changed my git installation for a while
2) Just tried running `git status`, works just fine.
3) I'm running Stitch using the NodeJS API. I am automatically downloading the sprites from Google Drive to then load them into the GMS2 project
4) I am running it natively, using VS Code's built-in terminal

Also I just tried running it and it somehow worked?? No idea what was that, but thanks for your help, Adam!
